backend/python/app/__init__.py

```diff
@@ -32,22 +33,29 @@ def create_app(config_name):
     )
 
     app = Flask(__name__)
+    mysql = MySQL()
     app.config.from_object(app_config[config_name])
 
     app.config["CORS_ORIGINS"] = ["http://localhost:3000"]
     app.config["CORS_SUPPORTS_CREDENTIALS"] = True
     CORS(app)
 
+    app.config["MYSQL_HOST"] = os.getenv("MYSQL_HOST")
+    app.config["MYSQL_USER"] = os.getenv("MYSQL_USER")
+    app.config["MYSQL_PASSWORD"] = os.getenv("MYSQL_PASSWORD")
+    app.config["MYSQL_DB"] = os.getenv("MYSQL_DB")
```

Comment:
`MYSQL_DATABASE` is actually what mysql will be looking for: https://hub.docker.com/_/mysql#Environment_Variables
FYI making this change in my commit